### ISSUE TITLE
amp-geo country code for the United Kingdom

### ISSUE THREAD

**Issue by fstanis** [ampproject/amphtml#15390]
## What's the issue?

Not sure if this was discussed before, but it seems that `amp-geo` returns "uk" as the country code for the United Kingdom. However, the actual country code for the UK, per [ISO 3166-1 alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2), is "GB", not "UK" (somewhat confusingly).

Main issue is that "GB" is already used by most other projects, so users may find AMPHTML incompatible with these (e.g. the locale is named "en_GB", not "en_UK").

Furthermore, it seems the documentation actually uses `amp-iso-country-gb` in the example:

https://github.com/ampproject/amphtml/blob/1e5ee8a46a90047e5e9e70983d0ece16f26d73e7/extensions/amp-geo/amp-geo.md#L97

## How do we reproduce the issue?

Sample code used (shows "UK" instead of "GB"):

```html
<!doctype html>
<html ⚡>
<head>
  <meta charset="utf-8">
  <title>Test</title>
  <script async src="https://cdn.ampproject.org/v0.js"></script>
  <script async custom-element="amp-geo" src="https://cdn.ampproject.org/v0/amp-geo-0.1.js"></script>
  <link rel="canonical" href="https://localhost:3000/">
  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
  <style amp-custom>
    .amp-iso-country-uk .country-code:before {
      content: "UK";
    }

    .amp-iso-country-gb .country-code:before {
      content: "GB";
    }
  </style>
  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
</head>
<body>
  <p class="country-code"></p>
  <amp-geo layout="nodisplay"></amp-geo>
</body>
</html>
```

## What browsers are affected?

Tested on Chrome desktop.

## Which AMP version is affected?

Tested on 1526498116488.

**Comment by jpettitt** [comment]
That's a bug it should be 'gb'.  I'll fix it.  Tracked internally b/80094782

**Comment by jpettitt** [comment]
Fixed cl/197465805 Fix will be live in the next hour.
